spec/granite/error/error_spec.cr

```diff
@@ -0,0 +1,8 @@
+require "../../spec_helper"
+
+describe Granite::Error do
+  it "should convert to json" do
+    error = Granite::Error.new "field", "error message"
+    expect error.to_json to equal "{\"field\": \"error message\"}"
```

Comment:
```suggestion
    Granite::Error.new("field", "error message").to_json.should eq %({"field":"error message"})
```

Reply:
Sorry, the tests don't like my local machine, had to push to test :P